lib/ansible/modules/crypto/openssl_certificate.py

```diff
@@ -534,7 +644,10 @@
 
 from random import randint
 import abc
+import base64
+import calendar
```

Comment:
They were necessary previously, for the ocsp check behavior and "renew" support that I since removed (to be in separate PR or different module entirely, as appropriate). I didn't remove them when I stripped the functionality out.

I'll remove them and also go through my code a second time to make sure there's nothing else hanging.